backend/helm/cmd_helm.go

```diff
@@ -47,6 +47,7 @@ func rollbackRelease(actionConfig *action.Configuration, name string, version in
 
 func uninstallRelease(actionConfig *action.Configuration, name string) (*release.UninstallReleaseResponse, error) {
 	uninstall := action.NewUninstall(actionConfig)
+	uninstall.Timeout = 300 * time.Second
```

Comment:
Does it need timeout? Can't it run on a completely new thread in the background and just finish when it finishes, or when the helm timeout lets up? Alternatively, I would set it, but for hours rather than minutes

Reply:
I'll check if this is needed for the uninstall, but it was necessary for the rollback to run at all.

Reply:
Ok, so timeout affects wheather or not helm will wait for all jobs and other resources to finish before deleting them. I'm not sure how important this is for us.